projects/ngx-datatable/src/lib/types/cell-context.type.ts

```diff
@@ -30,7 +30,7 @@ export interface CellContext<TRow> {
   isSelected: boolean;
   rowIndex: number;
   treeStatus: TreeStatus,
-  disable$: Observable<boolean>;
+  disable$: BehaviorSubject<boolean>;
```

Comment:
Why did you make this change? If there is no specific reason we could use the more generic interface.

Reply:
Our example expects this. So changing the type would break it. See: https://github.com/siemens/ngx-datatable/blob/master/src/app/basic/disabled-rows.component.ts

Reply:
This architectural concept is kind of questionable. But it is already in. 😳